shimmingtoolbox/read_nii.py

```diff
@@ -0,0 +1,77 @@
+import copy
+import json
+import math
+import nibabel as nib
+import numpy as np
+import os
+
+
+def read_nii(nii_path):
+    """ Reads a nifti file and returns image, info and json_data
```

Comment:
"returns image"
what is meant by '"image" here? ndarray? nibabel object? other?